File: C++/1431. Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& c, int e) {
        vector<bool>a;
        int m=*max_element(c.begin(),c.end());
        for(int i:c){
            a.push_back((i+e)>=m);
        }
        return a;
    }
};

File: JAVA/1431. Kids With the Greatest Number of Candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] c, int e) {
        List<Boolean> a=new ArrayList<>();
        int i,m=0;
        for(i=0;i<c.length;i++){
            if(m<c[i]){
                m=c[i];
            }
        }
        for(i=0;i<c.length;i++){
            a.add((c[i]+e)>=m);
        }
        return a;
    }
}